src/index.js

```diff
@@ -16,9 +18,24 @@ searchBtn.addEventListener('submit', (e) => {
       domPopulator(res, metricValue);
     })
     .catch(() => {
-      alert('location you entered is not valid.');
-      getTemp().then((res) => {
-        domPopulator(res, metricValue);
-      });
+      // alert('location you entered is not valid.');
+      document.querySelector('.alert').classList.add('show')
+      setTimeout(() => {
+      document.querySelector('.alert').classList.remove('show')
+      }, 3000)
+      document.querySelector('#searchInput').value = '';
+      getTemp().then(domPopulator);
     });
 });
+
+toggleBtn.addEventListener('click', () => {
+  console.log('hi');
```

Comment:
This should be a production code, please remove debugging statements. (`console.log`), even if that feature is not fully implemented, users should not see debugging messages.

I trust you could do this before merging.